src/docs/manual/method_list_example.txt

```diff
@@ -1,54 +1,54 @@
-method : java.util.AbstractSet.removeAll(java.util.Collection)
-method : java.util.Collections.binarySearch(java.util.List,java.lang.Object)
-method : java.util.Collections.binarySearch(java.util.List,java.lang.Object,java.util.Comparator)
-method : java.util.Collections.min(java.util.Collection)
-method : java.util.Collections.copy(java.util.List,java.util.List)
-method : java.util.Collections.enumeration(java.util.Collection)
-method : java.util.Collections.fill(java.util.List,java.lang.Object)
-method : java.util.Collections.list(java.util.Enumeration)
-method : java.util.Collections.max(java.util.Collection,java.util.Comparator)
-method : java.util.Collections.min(java.util.Collection)
-method : java.util.Collections.min(java.util.Collection,java.util.Comparator)
-method : java.util.Collections.reverseOrder()
-method : java.util.Collections.rotate(java.util.List,int)
-method : java.util.Collections.shuffle(java.util.List)
-method : java.util.Collections.singleton(java.lang.Object)
-method : java.util.Collections.singletonList(java.lang.Object)
-method : java.util.Collections.singletonMap(java.lang.Object,java.lang.Object)
-method : java.util.Collections.sort(java.util.List)
-method : java.util.Collections.sort(java.util.List,java.util.Comparator)
-method : java.util.Collections.swap(java.util.List,int,int)
-method : java.util.Collections.synchronizedCollection(java.util.Collection)
-method : java.util.Collections.synchronizedMap(java.util.Map)
-method : java.util.Collections.synchronizedSet(java.util.Set)
-method : java.util.Collections.synchronizedSortedSet(java.util.SortedSet)
-method : java.util.Collections.unmodifiableCollection(java.util.Collection)
-method : java.util.Collections.unmodifiableList(java.util.List)
-method : java.util.Collections.unmodifiableMap(java.util.Map)
-method : java.util.Collections.unmodifiableSet(java.util.Set)
-method : java.util.Collections.unmodifiableSortedSet(java.util.SortedSet)
-method : java.util.TreeSet.addAll(java.util.Collection)
-method : java.util.TreeSet.add(java.lang.Object)
-method : java.util.TreeSet.clear()
-method : java.util.TreeSet.clone()
-method : java.util.TreeSet.comparator()
-method : java.util.TreeSet.contains(java.lang.Object)
-method : java.util.TreeSet.first()
-method : java.util.TreeSet.headSet(java.lang.Object)
-method : java.util.TreeSet.isEmpty()
-method : java.util.TreeSet.iterator()
-method : java.util.TreeSet.last()
-method : java.util.TreeSet.remove(java.lang.Object)
-method : java.util.TreeSet.size()
-method : java.util.TreeSet.subSet(java.lang.Object,java.lang.Object)
+java.util.TreeSet()
+java.util.TreeSet(java.util.Collection)
+java.util.TreeSet(java.util.Comparator)
+java.util.TreeSet(java.util.SortedSet)
+java.lang.Object()
+java.util.AbstractCollection.containsAll(java.util.Collection)
+java.util.AbstractCollection.retainAll(java.util.Collection)
+java.util.AbstractCollection.toArray()
+java.util.AbstractCollection.toArray([Ljava.lang.Object;)
```

Comment:
The documentation of a fully-qualified signature does not indicate how to specify array types.  Please update the documentation.  It looks like you are using either the Class.getName format or field descriptiors, but not fully-qualified types which suggests use of the Java source code format; calling them fully-qualified names is misleading.  (I would guess you are using Class.getName.)